src/mainwindow.cpp

```diff
@@ -185,6 +187,17 @@ MainWindow::MainWindow(QWidget *parent) :
                 } else {
                     ui->lostMessage->setVisible(false);
                 }
+                ui->hostNameValue->setText(data.hostName);
+                ui->linuxKernelVersionValue->setText(data.linuxKernelVersion);
+                ui->perfVersionValue->setText(data.perfVersion);
+                ui->cpuDescriptionValue->setText(data.cpuDescription);
+                ui->cpuIdValue->setText(data.cpuId);
+                ui->cpuArchitectureValue->setText(data.cpuArchitecture);
+                ui->cpusOnlineValue->setText(QString::number(data.cpusOnline));
+                ui->cpusAvailableValue->setText(QString::number(data.cpusAvailable));
+                ui->cpuSiblingCoresValue->setText(data.cpuSiblingCores);
+                ui->cpuSiblingThreadsValue->setText(data.cpuSiblingThreads);
+                ui->totalMemoryValue->setText(format.formatByteSize(data.totalMemoryInKiB, 1, KFormat::MetricBinaryDialect));
```

Comment:
this is wrong, you need to pass in bytes, but totalMemoryInKiB, as the name indicates, is in KiB. So you'll have to multiply it by 1024.